getting-started/learn-the-basics.mdx

```diff
@@ -124,6 +124,37 @@ with open(data_path, "w") as f:
 
 </CodeGroup>
 
+## Delete Branch
+
+Once finished with a branch, you can delete it.
+
+<CodeGroup>
+
+```bash CLI
+# Checkout main branch locally
+oxen checkout main
+# Delete 'other_branch' locally
+oxen branch -d new_branch # may need -D if branch is not merged into main
+# Delete branch in remote repo
+oxen push origin --delete new_branch
+```
+
+```python Python
+import os
+from oxen import LocalRepo
+
+# Instantiate a LocalRepo object
+repo = LocalRepo("/path/to/data")
+# Checkout the main branch
+repo.checkout("main")
+# Delete new_branch
+repo.branch(name ='new_branch', delete=True)
```

Comment:
@Mathi0750 we can get rid of `name=` here right?

Reply:
on it

Reply:
done
btw, thanks for the rec of fish! it makes using my terminal so much nicer